server/server.go

```diff
@@ -2004,6 +2006,17 @@ func (s *Server) v1QueryPost(w http.ResponseWriter, r *http.Request) {
 	writer.JSON(w, 200, results, pretty)
 }
 
+func (s *Server) v1ConfigGet(w http.ResponseWriter, r *http.Request) {
+	pretty := getBoolParam(r.URL, types.ParamPrettyV1, true)
+	result, err := s.manager.Config.ActiveConfig()
+	if err != nil {
+		writer.ErrorAuto(w, err)
```

Comment:
`ErrorAuto` calls `types.NewErrorV1` or were you suggesting something like `fmt.Errorf("error getting active config: %w", err)` ?

Reply:
I think he's referring to the latter...I don't have a strong opinion here. I think the error cases from our own code in this path are unreachable (i.e., I don't see how those type assertions can fail assuming the config is valid, which if it's active, it better be!) and errors from json.Marshal/Unmarshal are also unlikely (again, because the config ought to be valid.)